### ISSUE TITLE
Add packer checkhealth

### ISSUE THREAD

**Issue by clason** [wbthomason/packer.nvim#525]
I haven't seen this explicitly suggested, so I am adding it here: I think it would be useful to add a `packer` section for Neovim's `:checkhealth`.

There's a discussion to be had about what exactly should be in there; some ideas:

* presence and location of a `packer_compiled.{lua,vim}`
* presence and version of luarocks installation
* paths for plugins?
* ...

**Comment by wbthomason** [comment]
Would "paths for plugins" be the package root, or a list of the installation paths for all your plugins?

Also, things to add: 
- Currently installed Luarocks rocks and their versions
- Timestamp for the last `packer.compile` run

**Comment by clason** [comment]
I was thinking "package root" (no sense duplicating `:PackerStatus`-- unless you want to fold that in?)

**Comment by wbthomason** [comment]
No, I think package root makes sense. I don't really want to duplicate `PackerStatus`, since I think most of that information is relevant to runtime state and not "health" (i.e. which plugins are loaded doesn't seem important for `checkhealth`).
